src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+
+  const stylesArray = stylesString
```

Comment:
res/result/el/elem/arr/array/obj/object and etc are bad names, variable name should be descriptive